config/manifests/bases/external-secrets-operator.clusterserviceversion.yaml

```diff
@@ -9,7 +9,12 @@ metadata:
     containerImage: ghcr.io/external-secrets/external-secrets-helm-operator:v0.8.1
     createdAt: "2021-11-22 00:00:00"
     description: Operator to configure external-secrets helm-chart based operator
+    operatorframework.io/cluster-monitoring: "true"
+    operators.openshift.io/infrastructure-features: '["Disconnected"]'
     repository: https://github.com/external-secrets/external-secrets-helm-operator
+  labels:
+    operatorframework.io/arch.amd64: supported
```

Comment:
Note to myself: docs about the labels are here: https://olm.operatorframework.io/docs/advanced-tasks/ship-operator-supporting-multiarch/#multiple-architectures